pandas/tests/groupby/test_apply.py

```diff
@@ -19,6 +19,26 @@
 from pandas.tests.groupby import get_groupby_method_args
 
 
+def test_apply_func_that_appends_group_to_list_without_copy():
+    # GH 17718
+
+    df = DataFrame(1, index=list(range(10)) * 10, columns=[0]).reset_index()
+    groups = []
+
+    def store(group):
+        groups.append(group)
+
+    df.groupby("index").apply(store)
+    expected_value = DataFrame(
```

Comment:
Could you make `expected` equal to 

```
DataFrame({"index": [0] * 10, 0: [1] * 10}, index=np.arange(0, 100, 10))
```

Reply:
> Could you make `expected` equal to
> 
> ```
> DataFrame({"index": [0] * 10, 0: [1] * 10}, index=np.arange(0, 100, 10))
> ```

No Problem ;)

Reply:
Hi,
this is my first time contributing on GitHub and I ran into some problems with failed checks while I was trying to update the pull request.
can you please help me with this problem?

> Could you make `expected` equal to
> 
> ```
> DataFrame({"index": [0] * 10, 0: [1] * 10}, index=np.arange(0, 100, 10))
> ```

Hi,
this is my first time contributing on GitHub and I ran into some problems with failed checks while I was trying to update the pull request.
can you please help me with this problem?

Reply:
> Could you make `expected` equal to
> 
> ```
> DataFrame({"index": [0] * 10, 0: [1] * 10}, index=np.arange(0, 100, 10))
> ```

Is it ok that I used pd.RangeIndex instead of np.arange? I got an error about dtypes and pd.RangeIndex fixed it